### DISCUSSION TITLE
Modern Toolbar

### DISCUSSION THREAD

**Discussion by Zer0-Official** [TomSchimansky/CustomTkinter#1492] [Ideas]
I don't know if someone has already made this, but I would like to see a modern toolbar for the top of Windows. These can include custom dropdowns like File, Edit, and Settings with a list of customizable features that call functions. I am new to Tkinter and therefore customtkinter, so please forgive me if it seems like I'm 8 years old here.

**Comment by Akascape** [comment]
If you want the toolbar in title bar area: https://github.com/TomSchimansky/CustomTkinter/discussions/1422#discussioncomment-5523331

If you want toolbar below the title bar: https://github.com/TomSchimansky/CustomTkinter/discussions/1422#discussioncomment-5522197

(Use the above dropdown class for making your modern dropdowns)
